aten/src/ATen/native/cuda/Utils.cuh

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <ATen/ATen.h>
+#include <ATen/native/cuda/Loops.cuh>
+#include <ATen/native/cuda/MemoryAccess.cuh>
+namespace {
+static constexpr int64_t kILP = 4;
+static constexpr int64_t kChunkSize = 65536;
+static constexpr int64_t kBlockSize = 1024;
```

Comment:
nit: @mcarilli recommends 512 here, as this is what they've been using in apex.

Reply:
512 gives the compiler a bit more leeway to use registers while satisfying a launch bound of 1 block per SM.